### PR TITLE
AACT-437: As a user I should be able to "view" the query

### PR DESCRIPTION
**Pull request by Javier-Jimenez-18**
- Added the show action to the SavedQueriesController. 
- Created the show view page:

Screen Shot 2022-11-02 at 11 03 32 AM

### CODE REVIEW

File: app/controllers/saved_queries_controller.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  def new
+    @saved_query = SavedQuery.new
+  end
+
+  def show
+  end
+
+  def create
+    @saved_query = SavedQuery.new(saved_query_params)
+    if @saved_query.save
```
Comment (micronix) [inline app/controllers/saved_queries_controller.rb:18]: We want to add the currently logged in user
```ruby
@saved_query.user_id = current_user.id
```

File: app/views/saved_queries/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <%= f.text_field :title, class: "form-control" %>
+    </div>
+    <div class="container-fluid">
+      <%= f.label :description %>
+      <%= f.text_area :description, rows: 5, class: "form-control" %>
+    </div>
+    <div class="container-fluid">
+      <%= f.label :sql %>
+      <%= f.text_area :sql, rows: 5, class: "form-control" %>
+    </div>
```
Comment (micronix) [inline app/views/saved_queries/_form.html.erb:24]: Missing `public` field

File: app/views/saved_queries/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          <th>Created</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @saved_queries.each do |query| %>
+          <tr>
+            <td><%= link_to query.title, saved_query_path(query) %> </td>
+            <td><%= query.description %></td>
+            <td><%= query.sql %></td>
+            <td><%= query.user %></td>
```
Comment (micronix) [inline app/views/saved_queries/index.html.erb:29]: put the user name instead of the user object

**Review by micronix — CHANGES_REQUESTED** [review]
Some small changes

**Comment by Javier-Jimenez-18** [comment]
- Updated create action. 
- Updated index view page to show username. 
- Updated _form view page to add a checkbox to make SQL query public. 
- Modified shared.scss stylesheet to remove label padding.

Screen Shot 2022-11-10 at 4 26 03 PM

Screen Shot 2022-11-10 at 4 26 44 PM

Screen Shot 2022-11-10 at 4 27 50 PM

Screen Shot 2022-11-10 at 4 28 27 PM

Screen Shot 2022-11-15 at 1 49 28 PM

**Comment by Javier-Jimenez-18** [comment]
In the Saved Queries show page, displayed “SQL Query is public” or “SQL Query is not public” (User clicks corresponding checkbox):

Screen Shot 2022-11-19 at 4 21 14 PM

Screen Shot 2022-11-19 at 4 21 38 PM

**Review by ievgend2 — APPROVED** [review]
Looks Good

**Pull request closed without merging**